policyengine_uk/tests/policy/baseline/gov/dfe/extended_childcare_entitlement/extended_childcare_entitlement_expenses.yaml

```diff
@@ -0,0 +1,137 @@
+- name: Eligible for 30 hours - All conditions met
+  period: 2025
+  input:
+    age: 3  # Age 3+ gets 30 hours
+    is_child: false
+    extended_childcare_entitlement_meets_income_requirements: true
```

Comment:
ensure these are true unit tests - so in this case using `extended_childcare_entitlement_eligible` instead and only other direct inputs. check other tests too (unless named `integration.yaml`)